doc/src/sgml/json.sgml

```diff
@@ -549,22 +568,36 @@ SELECT '"foo"'::jsonb ? 'foo';
     sub-objects.  This query finds entries in which sub-objects containing
     both <literal>"term":"paris"</> and <literal>"term":"food"</> appear,
     while ignoring any such keys outside the <literal>tags</> array:
+-->
+JSONはネストして含められるので、適切なクエリはサブオブジェクトの選択を明示的にスキップすることができます。
+例として、トップレベルに<structfield>doc</>列のオブジェクトが含まれ、<literal>tags</>フィールドにサブオブジェクトの配列が含まれているとします。
+この問い合わせは、配列の外側にあるようなキーを無視して、サブオブジェクトの<literal>"term":"paris"</>と<literal>"term":"food"</>を両方含まれるエントリを探し出すことが出来ます。
```

Comment:
松田さんのコメントは技術的にも正しいと思います。

あと、"Because JSON containment is nested"は、「JSONでは包含(関係)はネストしているので」という意味ではないでしょうか。このドキュメントの他の部分でも、"containment"は「包含」と訳しています。